service/controller/aws/v15/resource_set.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/giantswarm/operatorkit/controller"
 	"github.com/giantswarm/operatorkit/controller/resource/metricsresource"
 	"github.com/giantswarm/operatorkit/controller/resource/retryresource"
+	"github.com/giantswarm/operatorkit/resource/secret"
 	"github.com/giantswarm/tenantcluster"
 	"github.com/spf13/afero"
 	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

This was already there but lets take the opportunity to fix the alias.